locales/en.json

```diff
@@ -69,9 +68,11 @@
   "claimed": "Claimed",
   "closed": "Closed",
   "closedAuctionTitle": "Domain is no longer available",
+  "collapse": "Collapse",
```

Comment:
We should keep track of any new key added here, so we can add them to the other languages.

Reply:
Yeah it's hard to keep track of translations across PRs and languages. So thinking of this before every release:
- Filter all PRs with `needs translation` label like this one
- Translators then work on the master branch directly instead of each branch
- Make sure all languages are done before each release

Also, there are scripts in the `scripts` directory that compare langs and find missing fields (relative to en.json).